File: Assignment1/sumProd2Cons.cpp
#include <iostream>
using namespace std;

int main() {
    int n, sum = 0, a = 0;
    cout << "Enter how many numbers to enter: ";
    cin >> n;
    cout << "Enter " << n << " numbers: ";
    for (int i = 0; i < n; i++) {
        int x;
        cin >> x;
        sum += (a * x);
        a = x;
    }
    cout << sum << endl;
    return 0;
}

File: Assignment1/sumProd2Cons.java
import java.util.*;

public class sumProd2Cons
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter how many numbers to enter: ");
		int n = sc.nextInt();
		int sum = 0;
		int a = 0;
		System.out.print("Enter " + n + " numbers: ");
		for (int i=0; i<n; i++)
		{
			int x = sc.nextInt();
			sum += (a*x);
			a = x;
		}
		System.out.println(sum);

		sc.close();
	}
}